dotCMS/src/main/java/com/dotcms/browser/BrowserAPIImpl.java

```diff
@@ -375,13 +371,18 @@ private void includeFolders(final BrowserQuery browserQuery,
         
         pageMap.put("__icon__",      IconType.HTMLPAGE.iconName());
 
-        return pageMap;
+        //return pageMap;
+*/
+        final Map<String,Object> m = new DotContentletTransformer.Builder().webAssetOptions().content(page).build().toMaps().get(0);
+        System.out.println(m);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)